HTTPServer.driver/Driver.cs

```diff
@@ -32,6 +32,7 @@ public static void Main(string[] args)
             commandDetails.Add(Tuple.Create((ICriteria)new VersionNotSupportedCriteria(), (IHttpHandler)new VersionNotSupported()));
             commandDetails.Add(Tuple.Create((ICriteria)new ContentsCriteria(), (IHttpHandler)new GetContents(pathContents)));
             commandDetails.Add(Tuple.Create((ICriteria)new PostCriteria(), (IHttpHandler)new PostContents(pathContents)));
+            commandDetails.Add(Tuple.Create((ICriteria)new PutCriteria(), (IHttpHandler)new PutContents(pathContents)));
```

Comment:
That can be done.  Some of the details will remain in the Driver still.  Is that ok as long as it doesn't keep track of the details?